composer.json

```diff
@@ -9,7 +9,8 @@
     "require": {
         "php": ">=5.3.3",
         "symfony/symfony": "2.3.*",
-        "doctrine/orm": "~2.2,>=2.2.3",
+        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
+        "doctrine/dbal": "<2.5",
```

Comment:
@GrahamCampbell I think because of `doctrine/doctrine-bundle 1.2.*` it’s `>=2.2`: https://packagist.org/packages/doctrine/doctrine-bundle#v1.2.0